openrtb_ext/bidders.go

```diff
@@ -34,6 +28,40 @@ func (name *BidderName) String() string {
 	return string(*name)
 }
 
+// Names of reserved bidders. These names may not be used by a core bidder or alias.
+const (
+	BidderReservedAll     BidderName = "all"     // Reserved for the /info/bidders/all endpoint.
+	BidderReservedContext BidderName = "context" // Reserved for first party data.
+	BidderReservedGeneral BidderName = "general" // Reserved for non-bidder specific messages when using a map keyed on the bidder name.
+	BidderReservedSKAdN   BidderName = "skadn"   // Reserved for SKAdNetwork OpenRTB extension.
+	BidderReservedPrebid  BidderName = "prebid"  // Reserved for Prebid Server configuration.
+)
```

Comment:
Refactored reserved names. Added `skadn` since we'll be adding support really soon and I was adding `all` here already.